TOPIC8-MONGODB/src/main/java/data/CourseDao.java

```diff
@@ -0,0 +1,15 @@
+package data;
+
+import java.util.List;
+
+import models.Course;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.mongodb.repository.Query;
+
+public interface CourseDao extends MongoRepository<Course, Long> {
+
+	@Query(value = "{ $and [ { teacherAssigned.idTeacher: ?0 }, { courseName :?0 }}")

```

Comment:
Look this please http://docs.spring.io/spring-data/data-mongo/docs/1.4.0.RC1/reference/html/mongo.repositories.html (6.3 Query methods)